### PR TITLE
Update and rerender momentjs formatted values every second

### LINKED ISSUE #554: [Verified] Direct: message timestamp is not refreshed intermittently
It's expected that for visible in the feed/DM section messages timestamp will be refreshed as time passes.

Right now for some messages it randomly doesn't happen.
image

### PR DESCRIPTION
**Pull request by Nek**
closes #554
This looks like a hack but it makes sense much more than proposed
"idiomatic" way of fixing this one.
https://github.com/emberjs/ember.js/blob/680f997ed0958c420abdcd0b1673111aee26afe7/packages/ember-views/lib/mixins/normalized_rerender_if_needed.js

### CODE REVIEW

File: public/js/app/models/Post.js [hunk 1]
```diff
@@ -23,7 +24,15 @@ define(["config",
     isHidden: DS.attr('boolean'),
 
     timeline: DS.belongsTo('timeline'),
-
+    didLoad: function(){
+      setInterval(
+        Ember.run.bind(this, this.updateTime)
+        , 1000)
```
Comment (indeyets) [inline public/js/app/models/Post.js]: once a second? I'm not sure we need that kind of granularity here.
10 seconds should be just fine

(just asking what other ppl think)

Comment (epicmonkey) [inline public/js/app/models/Post.js]: Agree. In fact even 30s or 60s is alright

**Comment by Nek** [comment]
@berkus  Yes, plus does coercion to number. In case of a Date it returns it's timestamp.
That's used nearly everywhere as a polyfill for Date.now.

File: public/js/app/models/Post.js (hunk 1 above)
Comment (Nek) [inline public/js/app/models/Post.js]: ok
30s then

**Comment by clbn** [comment]
Yeah, 30s makes more sense.

I wonder if we can optimize it further by setting various interval depending on initial age. Like, it doesn't make much sense to update "1 month ago" property every 30 seconds. What do you guys think?

**Comment by clbn** [comment]
I'd still prefer some optimization: 1) interval depending on the age of post; and/or 2) using one app-wide counter. Our application is pretty bloated already, any slightest chance of making it slower/heavier makes me anxious.

**Comment by Nek** [comment]
There is a better way to deal with setInterval:
http://guides.emberjs.com/v1.10.0/cookbook/working_with_objects/continuous_redrawing_of_views/

**Comment by Nek** [comment]
This is very important:
UNCONSUMED COMPUTED PROPERTIES DO NOT TRIGGER OBSERVERS
http://guides.emberjs.com/v1.10.0/object-model/observers/

That's why there is:

```
init: function() {
      this._super()
      this.get('createdAt')
      this.get('currentTime')
    }
```

Without this part the newly created posts' time won't be updated at all.

**Comment by indeyets** [comment]
@epicmonkey, @clbn - review of updated code is needed here

**Comment by epicmonkey** [comment]
jftr: let's extract common parts to PostGenericController.

**Pull request merged**